README.md

```diff
@@ -113,111 +258,33 @@ An example configuration file is provided in `settings/default.json`:
 For more information about storage backends, please refer to the [Backends](src/storage/README.md) documentation,
 and for more information about reasoners, please refer to the [Reasoner](src/reasoner/README.md) documentation.
 
-### Launching
-
-Being a shared library, KnowRob cannot be launched directly but only in the context
-of a program that uses it.
 
-One such program is the terminal application that allows to interact with KnowRob
-using a command line interface.
-It can be launched as follows:
-
-```
-knowrob-terminal --config-file ~/knowrob/settings/default.json
-```
-
-The configuration file is a required argument, there is no fallback configuration file.
-
-Once the terminal is up and running, you should see a greeting message and a prompt
-which looks like this:
-
-```
-Welcome to KnowRob.
-For online help and background, visit http://knowrob.org/
-
-?- 
-```
+### Using the terminal application
 
 Please refer to the [CLI](src/queries/README.md) for documentation about the syntax of queries
 that can be typed into the terminal. Limited auto-completion is available. `exit/0` will
 terminate the terminal.
 
-Alternatively, you can expose the KnowRob querying interface as a ROS service.
-Please refer to the [ROS](src/ros/README.md) documentation for further information.
-
-## Getting Familiar
-
-Here we provide an overview about functionality of KnowRob.
-
-### Querying
-
-The core of KnowRob is a querying interface that is built around a custom querying language.
-Its syntax is similar to Prolog, but it is more simplified and not Turing-complete like Prolog is.
-
-For more information on querying in KnowRob, please have a look
-[here](src/queries/README.md).
-
-### Ontologies
+### Using the ROS interface
```

Comment:
I think this subsection should link to the subpage for the ros interface. You could then also have most of the client description there.